src/chat/components/button-icon/index.tsx

```diff
@@ -15,41 +16,25 @@ interface Props extends ComponentProps<'button'> {
 
 const ButtonBarAI: FC<Props> = ({
   icon,
-  className = '',
-  iconMainWrapClass,
-  active = false,
-  activeClass = 'bg-ch-fill-hover',
-  colorClass = 'text-ch-primary-gray2',
-
   tooltip,
   children,
   ...rest
 }) => {
-  const { t } = useTranslation();
   return (
-    <TooltipDefault title={t(tooltip)}>
-      <button
-        className={clsx([
-          className,
-          colorClass,
-          'hover:bg-ch-fill-hover disabled:text-ch-text-disabled flex h-6 items-center gap-[1px] rounded-lg p-1 transition-colors disabled:bg-transparent',
-          { [activeClass]: active },
-        ])}
-        type='button'
+    <Tooltip>
+      <TooltipTrigger asChild>
+      <Button
+        variant={'ghost'}
+        size={'icon'}
+        color={'default'}
```

Comment:
Icon's color does not match the design 
![image](https://github.com/user-attachments/assets/16d71483-ad42-443b-b707-ed3b65245e0d)